requirements.txt

```diff
@@ -1,11 +1,12 @@
-attrdict
-paramiko
+# The commented packages have been install to the docker slave by Dockerfile
 pytest==7.2.2
-pytest-services
-pytest-mock
-pytest-xdist
-requests
-unittest2
-urllib3
-PyYAML
-pexpect
+# attrdict
```

Comment:
Then one thing is we need to add some description for new hires or some other people new to this project about how to install the required package when testing locally or on the clean docker slave

Reply:
Redefined the requirements file and add back all the required packages, which could be used when we want to build a local slave to debug. 
And planning to remove the `pip install requirements.txt` step from the Jenkins Pipeline because all the packages should be already installed in the docker slave.